helm/aragorn-ranker/templates/env-config-map.yaml

```diff
@@ -11,4 +11,6 @@ data:
   OMNICORP_HOST: "{{ .Values.secrets.omnicorpHost }}"
   OMNICORP_PASSWORD: "{{ .Values.secrets.omnicorpPassword }}"
   OMNICORP_PORT: "{{ .Values.omnicorpPostgres.port }}"
-  OMNICORP_USER: "{{ .Values.omnicorpPostgres.dbUser }}"
\ No newline at end of file
+  OMNICORP_USER: "{{ .Values.omnicorpPostgres.dbUser }}"
+  AC_CACHE_HOST: "{{ .Values.answercoalesce.fullName }}-redis-service"
```

Comment:
I think this assumes `-redis-service` to always be part of the host name, which i agree is fine for us but in a more generic way I think we should provide  `{{ .Values.answercoalesce.fullName }}` so  we can support any redis  host (?)